src/app/client/src/app/modules/shared-feature/components/tnc-popup/terms-conditions-popup.component.ts

```diff
@@ -75,7 +75,7 @@ export class TermsAndConditionsPopupComponent implements OnInit, OnDestroy {
       }
     };
     if (this.userService.userProfile.managedBy) {
-      requestBody.request['userId'] = this.userService.userProfile.managedBy;
+      requestBody.request['userId'] = this.userService.userid;
```

Comment:
@deveshMantra please review this segment.

Reply:
Its okay as in release-3.0.0 we are using same code seems like 3.0.0 is not merged with 3.1.0.
Sending managed-user id for TNC accept and not sneding user id if it is parent user.